src/bin/add/args.rs

```diff
@@ -104,6 +104,15 @@ pub struct Args {
     /// Keep dependencies sorted
     #[structopt(long = "sort", short = "s")]
     pub sort: bool,
+
+    /// Registry to use
+    #[structopt(
+        long = "registry",
+        conflicts_with = "git",
+        conflicts_with = "path",
+        conflicts_with = "build"
```

Comment:
Are you sure it conflicts with build dependencies? Seems to work locally and I couldn't find it shouldn't in the RFC.
```suggestion
```

Reply:
I'm mistaken here, so will remove this, thanks!